### PR TITLE
Replacing status bar spinner

### LINKED ISSUE #505: Replace the spinner for `initial full test run`
### Expected Behavior

Status bar will be static, and not jumpy...

### Actual Behavior

The current slash spinner cause entire status bar to jump since the length of the spinner frames are different. This is very annoying...

--- 

I think I can send a PR that is using the [VS Code spinner](https://code.visualstudio.com/api/references/icons-in-labels#animation) `$(sync~spin)`

### PR DESCRIPTION
**Pull request by rhalaly**
Replacing the status bar spinner with the [VS Code spinner](https://code.visualstudio.com/api/references/icons-in-labels#animation) `$(sync~spin)`.

This change will make the status bar to be less "jumpy" since the old slashes spinner frames have no equal width.

Fixes #505

### CODE REVIEW

**Comment by rhalaly** [comment]
@connectdotz Since 100.0% added relevant lines in are covered, and no unchanged relevant lines lost coverage.

Can you please assist?

File: CHANGELOG.md [hunk 1]
```diff
@@ -12,6 +12,10 @@ Bug-fixes within the same version aren't needed
 
 -->
 
+## Master
+
+* replacing the status bar spinner to VS Code spinner
```
Comment (connectdotz) [inline CHANGELOG.md]: you might want to add user name at the end...

**Comment by rhalaly** [comment]
Thank you! Yes, this is the reason I asked you to step in :). I will resolve the comment you left and complete the PR.

File: CHANGELOG.md (hunk 1 above)
Comment (rhalaly) [reply CHANGELOG.md]: Fixed

**Pull request merged**